src/cli/search.rs

```diff
@@ -220,9 +225,16 @@ where
         false,
     )
     .await?;
+
+    if packages.is_empty() {
+        let normalized_package_name = package_name_search.as_normalized();
+        return Err(miette::miette!("Package {normalized_package_name} not found, please use a wildcard '*' in the search name for a broader result."));
```

Comment:
Does that actually work?

Reply:
If yes, please add a test for it :)

Reply:
Took care of this, it works and it's now tested in CI 👍